# Request
float64 angle
---
# Response